daffodil-japi/src/test/java/org/apache/daffodil/example/TestJavaAPI.java

```diff
@@ -984,7 +984,7 @@ public void testJavaAPI18() throws IOException, ClassNotFoundException {
    * daffodil/daffodil-sapi/src/test/resources/test/sapi
    *
    */
-    @Test
+    /*@Test*/
```

Comment:
Instead of commenting these out, can we just use a schema that will throw a Runtime SDE someway other than excessive backtracking?

Reply:
Done